core-bundle/src/Resources/contao/widgets/SerpPreview.php

```diff
@@ -42,8 +44,18 @@ public function generate()
 		$description = StringUtil::substr($this->getDescription($model), 160);
 		$alias = $this->getAlias($model);
 
-		// Get the URL with a %s placeholder for the alias or ID
-		$url = $this->getUrl($model);
+		try
+		{
+			// Get the URL with a %s placeholder for the alias or ID
+			$url = $this->getUrl($model);
+		}
+		catch (InvalidParameterException $exception)
```

Comment:
You should catch the `Symfony\Component\Routing\Exception\ExceptionInterface` to catch any routing exceptions.

Reply:
Done in 9a53820.